.circleci/config.yml

```diff
@@ -3,18 +3,15 @@
 # To manually manage the CircleCI configuration for this project, remove the .circleci/template.sh file.
 
 version: 2
-
-build-defaults: &build-defaults
-  resource_class: xlarge
-  environment:
-    GRADLE_OPTS: -Dorg.gradle.console=plain -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false
-    CIRCLE_TEST_REPORTS: /home/circleci/junit
-    CIRCLE_ARTIFACTS: /home/circleci/artifacts
-
 jobs:
   build:
-    docker: [{ image: 'circleci/openjdk:8u181-node' }]
-    <<: *build-defaults
+    docker: [{ image: 'circleci/openjdk:8u171-node' }]
```

Comment:
holding off on merging because this seems off to revert back to 8u171.

Reply:
going to approve for now as the rollback from 8u181 to 8u171 is to work around broken ALPN tests in other projects that don't affect tritium